buildSrc/src/main/kotlin/cleanup.gradle.kts

```diff
@@ -0,0 +1,105 @@
+check(rootProject.name == name) {
+    "The cleanup plugin should be applied to the root project and not $name"
+}
+
+tasks.register("templateCleanup") {
+    val repository = System.getenv("GITHUB_REPOSITORY")
+        ?: error("No GITHUB_REPOSITORY environment variable. Are you running from Github Actions?")
+
+    val (owner, name) = repository.split("/").let {
+        it[0].sanitized() to it[1].sanitized()
+    }
+
+    file("settings.gradle.kts").replace(
+        "rootProject.name = (\"kotlin-android-template\")",
+        "rootProject.name = (\"$name\")"
+    )
+    file("buildSrc/src/main/java/Coordinates.kt").replace(
+        "com.ncorti.kotlin.template",
+        "com.github.$owner.$name"
+    )
+    /**
+     * This fails with: 'refusing to allow a GitHub App to create or update workflow
+     * `.github/workflows/publish-release.yaml` without `workflows` permission'
```

Comment:
I think it's ok to remove the check at https://github.com/cortinico/kotlin-android-template/blob/master/.github/workflows/publish-release.yaml#L10 and enable the publishing workflows by default for all forks. If the secrets are not setup, upload will fail but I think that's ok and this gives some visibility to the publishing workflows.

Reply:
The reason why I initially disabled it was to make sure that once you start using the template the CI is green and you don't have to _understand_ why it fails.

Reply:
Makes sense. So maybe we leave these files untouched and add a comment?

```
jobs:
  publish:
    # remove this check when your secrets are setup and you're ready to publish
    if: ${{ github.repository == 'cortinico/kotlin-android-template'}}
```

Reply:
Agree